README.md

```diff
@@ -65,6 +66,9 @@ Below is a list of environment variables that will affect the vignette runtime.
  * `ENABLE_ACCESS_LOG`            enable the NCSA access log [false]
  * `ACCESS_LOG_FILE`              NCSA acces log file [/tmp/Vignette-access.log]
  * `IMAGEMAGICK_BASE`             path to the root of the ImageMagick installation [/usr/local]
+ * `CONSUL_HOSTNAME`              Consul Agent's address [localhost]
+ * `CONSUL_HTTP_PORT`             Consul Agent's port [8500]
+ * `CONSUL_QUERY_TAG`             Default tag that will be used to query instances [production]
```

Comment:
We're launching services with `prod` now, has that been done for the static asset service?